src/AppContainer.react.js

```diff
@@ -30,9 +30,10 @@ class UnconnectedAppContainer extends React.Component {
         if (type(config) === 'Null') {
             return <div className="_dash-loading">Loading...</div>;
         }
+        const {show_undo_redo} = config;
         return (
             <React.Fragment>
-                <Toolbar />
+                {show_undo_redo ? <Toolbar /> : null}
```

Comment:
Maybe a note in the Changelog to the effect that that the CSS solution is now obsolete.

Reply:
changelog: https://github.com/plotly/dash-renderer/pull/175/commits/e1941e1a468c9abb6d2d12c18c54a19324c6283e